src/Output.cs

```diff
@@ -2,124 +2,133 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Text;
+using Newtonsoft.Json;
+using System.Linq;
 
 namespace ACConfigBuilder
 {
     public class Output
     {
-        public void getobject(ACConfig AC, string path)
+        public dynamic writeOutput(List<string> AC, dynamic s)
         {
-            List<string> configback = new List<string>();
-            if (AC.configureNetwork != null)
+            StreamWriter writer = new StreamWriter(s);
+            using (writer)
             {
-                if (AC.configureNetwork.networkdev != null)
-                {
-                    configback.Add("configure network");
-                    foreach (var item in AC.configureNetwork.networkdev)
-                    {
-                        foreach (var propertyInfo in item.GetType().GetProperties())
-                        {
-                            var value = propertyInfo.GetValue(item);
-                            if (value != null)
-                            {
-                                if (value.ToString() == "True")
-                                {
-                                    configback.Add("  activate");
-                                }
-                                else
-                                {
-                                    configback.Add("  " + propertyInfo.Name + " " + value);
-                                }
-                            }
-                        }
-                        configback.Add(" exit");
-                    }
-                }
-                if (AC.configureNetwork.interfacenetworkif != null)
+                foreach (var item in AC)
                 {
-                    foreach (var item in AC.configureNetwork.interfacenetworkif)
-                    {
-                        foreach (var propertyInfo in item.GetType().GetProperties())
-                        {
-                            var value = propertyInfo.GetValue(item);
-                            if (value != null)
-                            {
-                                if (value.ToString() == "True")
-                                {
-                                    configback.Add("  activate");
-                                }
-                                else
-                                {
-                                    configback.Add("  " + propertyInfo.Name + " " + value);
-                                }
-                            }
-                        }
-                        configback.Add(" exit");
-                    }
-                    configback.Add("exit");
+                    writer.WriteLine(item);
+                    writer.Flush();
                 }
             }
+            return s;
+        }
+        public void  writeOutput(List<string> AC, string path)
+        {
+            writeOutput(AC, path);
```

Comment:
hier würde ich dann folgendes machen:

```suggestion
            using(var stream = new StreamWriter(path)){
               writeOutput(AC, stream);
            }
```